scripts/generate-docs-checkout.ts

```diff
@@ -18,6 +18,7 @@ const checkout = {
   outputRoot: '../shopify-dev/content/api/checkout-extensions/checkout',
   shopifyDevUrl: '/api/checkout-extensions/checkout',
   shopifyDevAssets: '../shopify-dev/app/assets/images/api/checkout-extensions',
+  shopifyDevAssetsUrl: '/assets/api/checkout-extensions',
```

Comment:
Let's keep the component assets for checkout-ui and the api assets for checkout-ui in the same location. `/assets/api/checkout-extensions/checkout`